<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bancaire</class>
 <widget class="QDialog" name="bancaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QComboBox" name="cartes">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>580</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="download">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="icon1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 0, 0);
font: 195 22pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="ag">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agence</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(220, 220, 220);</string>
   </property>
   <property name="text">
    <string>13/05/2020 - 13/06/2020</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(78, 214, 255);
font: 63 italic 14pt &quot;Source Code Pro Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>La Banque</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;OpenSymbol&quot;;</string>
   </property>
   <property name="text">
    <string>Produits Bancaires</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readOut">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>131</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="print">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="home">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(50, 0, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="prets">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comptes">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mode">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>11</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(158, 158, 158);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="chequiers">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>580</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>1141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="agence">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>191</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="valide1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valide2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valide3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valide4">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>valider</string>
   </property>
  </widget>
  <widget class="QComboBox" name="file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="view">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="help">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="pret">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>380</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="compte">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="chequier">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="carte">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>380</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <zorder>cartes</zorder>
  <zorder>download</zorder>
  <zorder>label_3</zorder>
  <zorder>label_8</zorder>
  <zorder>ag</zorder>
  <zorder>date</zorder>
  <zorder>label_2</zorder>
  <zorder>label_9</zorder>
  <zorder>readOut</zorder>
  <zorder>print</zorder>
  <zorder>home</zorder>
  <zorder>prets</zorder>
  <zorder>comptes</zorder>
  <zorder>mode</zorder>
  <zorder>chequiers</zorder>
  <zorder>cal</zorder>
  <zorder>line_2</zorder>
  <zorder>line</zorder>
  <zorder>agence</zorder>
  <zorder>icon1</zorder>
  <zorder>valide1</zorder>
  <zorder>valide2</zorder>
  <zorder>valide3</zorder>
  <zorder>valide4</zorder>
  <zorder>file</zorder>
  <zorder>view</zorder>
  <zorder>edit</zorder>
  <zorder>help</zorder>
  <zorder>calendar</zorder>
  <zorder>pret</zorder>
  <zorder>compte</zorder>
  <zorder>chequier</zorder>
  <zorder>carte</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
